app/frontend/src/components/admin/AdminVersions.vue

```diff
@@ -120,7 +147,7 @@ export default {
         this.formSchema = { ...this.formSchema, ...res.data.schema };
       } catch (error) {
         this.addNotification({
-          message: 'An error occurred while loading the form design.',
+          message: this.message,
```

Comment:
Why not just use the translation here instead of declaring a variable that isn't descriptive for it?

Reply:
I did not declare the variable. It was already declared, I just did the translation

Reply:
We should use the direct translation here regardless.